.github/workflows/test-software.eessi.io.yml

```diff
@@ -2,11 +2,25 @@
 name: Check for missing software installations in software.eessi.io
 on:
   push:
-    branches: [ "*-software.eessi.io" ]
+#    branches: [ "*-software.eessi.io" ]
+    branches: "*"
   pull_request:
   workflow_dispatch:
 permissions:
   contents: read # to fetch code (actions/checkout)
+env:
+  EESSI_ACCELERATOR_TARGETS: |
+    aarch64/generic:
+    aarch64/neoverse_n1:
+    aarch64/neoverse_v1:
```

Comment:
I would remove entries for which we don't have accelerator builds yet (and likely never will, like haswell)

Reply:
Done in https://github.com/EESSI/software-layer/pull/753/commits/4a5808c5f343b34190b0fd050023bd892ce23928.